web_modules/portal/src/components/page/CopyrightComponent.jsx

```diff
@@ -67,18 +67,28 @@ export class CopyrightComponent extends React.Component {
 
   getLayerCopyrights = layer => (
     <TableRow key={layer.id}>
+     // Does not display the layer name when no attribution
```

Comment:
Pas sûr que ce commentaire soit bien compris comme étant un commentaire...! Je m'attend plus à 
{* Does not display the layer name when no attribution *}